core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -656,9 +698,9 @@ public void close() throws Exception {
    * @param pageInfo page info
    * @return true if successful, false otherwise
    */
-  private boolean deletePage(PageInfo pageInfo) {
+  private boolean deletePage(PageInfo pageInfo, boolean isTemporary) {
     try {
-      pageInfo.getLocalCacheDir().getPageStore().delete(pageInfo.getPageId());
+      pageInfo.getLocalCacheDir().getPageStore().delete(pageInfo.getPageId(), isTemporary);
     } catch (IOException | PageNotFoundException e) {
       LOG.error("Failed to delete page {} from pageStore", pageInfo.getPageId(), e);
```

Comment:
Since you are adding the 'isTemporary', would it be useful to add that parameter to the error log?